### PR TITLE
fix/compile

### PR DESCRIPTION
**Pull request by sebascm**
## Compile errors

- Javax.inject dependency needed
    - Added to pom
- Warning with deprecated class in DataMapper
    - Couldn't fix the warning. I've updated the dependencies associated to the class deprecated but didn't work. I've put a SuppressWarning

### CODE REVIEW

**Comment by franrua** [comment]
At least 2 reviews must be required, as I say in [documentation](https://gitlab.com/torus_fitx/reference/blob/master/projects/henhouse.md). Please, read it. And don't merge this branch until you have approval from @Chusca.

**Review by franrua — APPROVED** [review]

**Review by Chusca — APPROVED** [review]

**Pull request merged**
